corehq/apps/export/static/export/js/export_list.js

```diff
@@ -183,21 +183,23 @@ hqDefine("export/js/export_list", [
 
         // Loading/error handling UI
         self.loadingErrorMessage = ko.observable('');
-        self.isLoading = ko.observable(true);
+        self.isLoadingPanel = ko.observable(true);
+        self.isLoadingPage = ko.observable(false);
         self.hasError = ko.observable(false);
         self.showError = ko.computed(function () {
-            return !self.isLoading() && self.hasError();
+            return !self.isLoadingPanel() && self.hasError();
         });
         self.showEmpty = ko.computed(function () {
-            return !self.isLoading() && !self.hasError() && !self.exports().length;
+            return !self.isLoadingPanel() && !self.hasError() && !self.exports().length;
         });
         self.showPagination = ko.computed(function () {
-            return !self.isLoading() && !self.hasError() && self.exports().length;
+            return !self.isLoadingPanel() && !self.hasError() && self.exports().length;
```

Comment:
does "panel" refer to the table / data that is loading?

Reply:
yeah, panels are these things: https://getbootstrap.com/docs/3.3/components/#panels-heading